shoes-core/lib/shoes/ui/picker.rb

```diff
@@ -21,15 +26,47 @@ def bundle
       end
 
       def select_generator
-        puts "Selecting Shoes backend to use. This is a one-time operation."
         candidates = Gem.find_files("shoes/**/generate-backend.rb")
         if candidates.one?
-          generator_file = candidates.first
+          candidate = candidates.first
+          @output.puts "Selecting #{name_for_candidate(candidate)} backend to use. This is a one-time operation."
+          candidate
         else
-          fail NotImplementedError("Currently can't interactively select a backend. See #929")
+          output_candidates(candidates)
+          prompt_for_candidate(candidates)
         end
+      end
+
+      def output_candidates(candidates)
+        @output.puts
+        @output.puts "Select a Shoes backend to use (This is a one-time operation):"
```

Comment:
Might not be a bad idea, even with the message after a failure. I did pattern the styling after the `gem uninstall` commands, so it might be familiar to some, but no sense making it more cryptic than necessary.